src/lib-voyager.tsx

```diff
@@ -44,7 +44,7 @@ export class Voyager {
       this.container = container;
     }
 
-    this.config = config;
+    this.config = Object.assign({}, DEFAULT_VOYAGER_CONFIG, config);
```

Comment:
`{...DEFAULT_VOYAGER_CONFIG, ...config}`

(We use the spread operator in this project.)